version.go

```diff
@@ -35,6 +35,16 @@ func (version *ArtifactVersion) refresh() {
 	}
 }
 
+// isNightlyOrContinuous returns true if the version doesn't indicate an official release.
+func (version *ArtifactVersion) isNightlyOrContinuous() (bool, error) {
+	parsedVersion, err := CreateMongoDBVersion(version.Version)
+	if err != nil {
+		return false, errors.Wrap(err, "could not parse version")
+	}
+	// handle legacy and modern versioning schemes
+	return !parsedVersion.IsStableSeries() || parsedVersion.IsContinuous(), nil
```

Comment:
Won't `IsStableSeries` always return true for new versions? I think checking for `parsedVersion.IsDevelopmentSeries()` probably makes more sense to handle the legacy versions.

Also not certain about including continuous releases in this, because of my comment above.

Reply:
`IsStableSeries` always returns false for the new versions.

Reply:
IsStableSeries will always return false for new versions, because it's applicable only to legacy. IsDevelopmentSeries also always returns false for new versions, so either is fine, but you're right that we might as well use that here instead of !IsStable.